stl/src/vector_algorithms.cpp

```diff
@@ -5,7 +5,7 @@
 // injected into the msvcprt.lib and msvcprtd.lib import libraries.
 // Do not include or define anything else here.
 // In particular, basic_string must not be included here.
-#if (defined(_M_IX86) || defined(_M_X64)) && !defined(_M_CEE_PURE)
+#if (defined(_M_IX86) || defined(_M_X64) && !defined(_M_ARM64EC)) && !defined(_M_CEE_PURE)

```

Comment:
I'm vaguely surprised that this line doesn't require a `!defined(_M_HYBRID)`. Maybe I'm missing something in my mental model on how ARM64EC is supposed to work.

Reply:
I think the reason is that `vector_algorithms.cpp` was added to the list of files to compile for EC, but I bet it's not being compiled at all for Hybrid. (My understanding of these is near-zero.) I can investigate this as a cleanup for a later PR.